hack/install-deps.sh

```diff
@@ -146,5 +146,6 @@ ${sudo} bash -c 'cat >'${CRICTL_CONFIG_DIR}'/crictl.yaml <<EOF
 runtime-endpoint: /var/run/cri-containerd.sock
 EOF'
 
-# Clean the tmp GOPATH dir
-${sudo} rm -r ${GOPATH}
+# Clean the tmp GOPATH dir. Use sudo because runc build generates
+# some privileged files.
+sudo rm -r ${GOPATH}
```

Comment:
did you mean to remove {} from here and above?

Reply:
Added `${sudo}` back, use `-rf` instead.